tf/edgeml/trainer/bonsaiTrainer.py

```diff
@@ -363,14 +390,16 @@ def train(self, batchSize, totalEpochs, sess,
                             self.isDenseTraining is False)):
                     self.runSparseTraining(sess)
                     if counter == int(2 * totalBatches / 3.0):
-                        msg = " Sprase Retraining Phase Started "
+                        msg = " Sparse Retraining Phase Started "
                         print("\n%s%s%s\n" %
                               (header, msg, header), file=self.outFile)
                 counter += 1
-
-            print("Train Loss: " + str(trainLoss / numIters) +
-                  " Train accuracy: " + str(trainAcc / numIters),
-                  file=self.outFile)
+            try:
```

Comment:
Print something for regression as well. We want to see the training pattern for regressio.